distributed/tests/test_scheduler.py

```diff
@@ -1845,6 +1845,37 @@ async def test_get_task_duration(c, s, a, b):
         assert len(s.unknown_durations["slowinc"]) == 1
 
 
+@gen_cluster(client=True)
+async def test_default_task_duration_splits(c, s, a, b):
```

Comment:
Hmm this test seems to raise `TimeoutError`s in a few CI builds

Reply:
I'm pretty sure this is connected to improper cluster shutdowns I've been observing recently. I am using almost identical code for the stealing test with the exception that I'm not awaiting the computation here. I'll add a wait in here and hope this is gone afterwards